using_images/xpaas_images/a_mq.adoc

```diff
@@ -105,6 +115,61 @@ application's Git project root. On each commit, the file will be copied to the
 *_conf_* directory in the A-MQ root and its contents used to configure the
 broker.
 
+== Configuring the JBoss A-MQ Persistance Image
```

Comment:
So to be clear, I think it should be called "the JBoss A-MQ Persistent Image" here.